mintapi/api.py

```diff
@@ -530,29 +550,32 @@ def get_transactions_json(self, include_investment=False,
         # Converts the start date into datetime format - must be mm/dd/yy
         try:
             start_date = datetime.strptime(start_date, '%m/%d/%y')
-        except:
+        except (TypeError, ValueError):
             start_date = None
         all_txns = []
         offset = 0
         # Mint only returns some of the transactions at once.  To get all of
         # them, we have to keep asking for more until we reach the end.
         while 1:
+            url = MINT_ROOT_URL + '/getJsonData.xevent'
+            params = {
+                'queryNew': '',
+                'offset': offset,
+                'comparableType': '8',
+                'rnd': Mint.get_rnd(),
```

Comment:
This looks great! But what happens to query_options?

Reply:
Ah I see, it ends up as params :)

Reply:
Yup! the requests lib does all the url encoding instead :) Way cleaner and less error prone. This lib was already using params elsewhere, so this is nice to start unifying.